master/buildbot/db/users.py

```diff
@@ -28,21 +28,23 @@ class UsDict(dict):
 class UsersConnectorComponent(base.DBConnectorComponent):
     # Documentation is in developer/db.rst
 
+    def __init__(self, *args, **kwargs):
+        super(UsersConnectorComponent, self).__init__(*args, **kwargs)
+        self._tbl = self.db.model.users
```

Comment:
we are never using this design pattern in any db modules, so I dont think we should do it here.